src/app/ApplicationTemplate.UWP/Views/Styles/Controls/PathControl.xaml

```diff
@@ -57,7 +57,7 @@
 		<Setter Property="Template">
 			<Setter.Value>
 				<ControlTemplate TargetType="u:PathControl">
-					<Path Fill="{TemplateBinding Foreground}"
+					<Path Fill="{StaticResource MaterialOnPrimaryMedEmphasisBrush}"
```

Comment:
for this file, wouldn't it be preferable to leave `TemplateBinding` and set the StaticResource via a setter on `Foreground` ?